shell/platform/common/cpp/BUILD.gn

```diff
@@ -68,7 +68,7 @@ source_set("common_cpp") {
   deps = [
     ":common_cpp_library_headers",
     "//flutter/shell/platform/common/cpp/client_wrapper:client_wrapper",
-    "//flutter/shell/platform/embedder:embedder_as_internal_library",
+    # "//flutter/shell/platform/embedder:embedder_as_internal_library",
```

Comment:
Why should you leave it as a comment?

Reply:
It doesn't much matter but to clearly differentiate the changed portion of the code so that it's more readable and easy to track.